Automattic/wp-calypso
# Hosting Metrics: Add a time range picker

See https://github.com/Automattic/dotcom-forge/issues/3255

## Proposed Changes

This PR adds a time and date picker to the `site-monitoring` page. The picker includes the options for 6 hours, 24 hours, 3 days and 7 days. 

## Testing Instructions

* Make sure you have an Atomic site
* Navigate to Calypso Live link (or pull this branch)
* Navigate to `site-monitoring/{yourAtomicSite}` (replace Atomic Site with an actual site's domain)
* Observe the time-date picker and confirm it defaults to 24 hours:

<img width="908" alt="Screenshot 2023-08-03 at 3 11 24 PM" src="https://github.com/Automattic/wp-calypso/assets/25575134/c9c99bff-57d8-4e42-b259-be58786ad329">

* Switch the options to `6 hours`, `24 hours`, `3 days` and `7 days` and confirm that the graph is updating and the display is changing

**Known issues**:

* the graph only displays hours so far and not dates - I will handle the desired date format in the future PR;
* the graphs on mobile do not look good (looks okay on tablets) so we might want to handle it in the future
## Pre-merge Checklist
